eclipse/src/saros/ui/ImageManager.java

```diff
@@ -78,8 +78,13 @@
       getImage("icons/elcl16/contact_add_tsk.png"); // $NON-NLS-1$
   public static final Image DLCL_CONTACT_ADD =
       getImage("icons/dlcl16/contact_add_tsk.png"); // $NON-NLS-1$
+
   public static final Image ELCL_SESSION = getImage("icons/elcl16/session_tsk.png"); // $NON-NLS-1$
+  public static final ImageDescriptor ELCL_SESSION__DESCRIPTOR =
```

Comment:
Just a question: Do you know the meaning of the abbreviations (ELCL, DLCL)?

Reply:
See https://wiki.eclipse.org/User_Interface_Guidelines Section Folder Structure